### PR TITLE
Added CultureInfo extension method ToLanguage (#357)

### PR DESCRIPTION
**Pull request by JanOuborny**
Added `CultureInfo` extension method `ToLanguage `and corresponding unit tests

### CODE REVIEW

File: RiotSharp/Misc/CultureInfoExtensions.cs [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    {
+        /// <summary>
+        /// Converts the value of this instance to its equivalent Language representation.
+        /// </summary>
+        /// <param name="cultureInfo"></param>
+        /// <returns>The Language representation of the value of this instance.</returns>
+        /// <exception cref="ArgumentException">Thrown if an unsupported CultureInfo is provided.</exception>
+        public static Language ToLanguage(this CultureInfo cultureInfo)
+        {
+            if (cultureInfo.IsNeutralCulture) // Neutral cultures don't have an equivalent in Riot's API, therefore they need to be mapped to a default
```
Comment (BenFradet) [inline RiotSharp/Misc/CultureInfoExtensions.cs]: could you add the comment above the line, it'll be easier to read

File: RiotSharp/Misc/CultureInfoExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            {
+                var riotLanguageName = cultureInfo.Name.Replace('-', '_');
+                Language language = Language.en_US;
+                if (Enum.TryParse(riotLanguageName, out language))
+                {
+                    return language;
+                }
+                else
+                {
+                    return ParseNeutralCulture(cultureInfo.Parent); // Parent culture is always neutral (as far as the supported languages go)
```
Comment (BenFradet) [inline RiotSharp/Misc/CultureInfoExtensions.cs]: same

File: RiotSharp/Misc/CultureInfoExtensions.cs [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+                case "th":
+                    return Language.th_TH;
+                case "tr":
+                    return Language.tr_TR;
+                case "vn":
+                    return Language.vn_VN;
+                case "zh":
+                    return Language.zh_CN;
+                default:
+                    throw new ArgumentException("Unsupported language:" + cultureInfo.Name);
```
Comment (BenFradet) [inline RiotSharp/Misc/CultureInfoExtensions.cs]: missing a space after `language: "`

File: RiotSharpTest/Misc/CultureInfoExtensionsTest.cs [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+                    // Act
+                    var language = directlySupportedCulture.ToLanguage();
+                }
+                catch
+                {
+                    // Assert
+                    Assert.Fail("Conversion of " + directlySupportedCulture.Name + " failed.");
+                }
+            }
+        }
```
Comment (BenFradet) [inline RiotSharpTest/Misc/CultureInfoExtensionsTest.cs]: why aren't you defining a map of `CultureInfo` -> `Language` and asserting the result of the conversion?

Comment (JanOuborny) [inline RiotSharpTest/Misc/CultureInfoExtensionsTest.cs]: The current solution was quicker to write. However, you are right, I am not asserting the result. I'm only making sure that every supported `CultureInfo `gets successfully converted into any `Language`, not in the correct `Language`. I'm gonna improve that.

Comment (BenFradet) [inline RiotSharpTest/Misc/CultureInfoExtensionsTest.cs]: great :+1:

**Comment by BenFradet** [comment]
Merging, thanks a lot!

**Comment by JanOuborny** [comment]
Your welcome. Expect some more pull requests today.

**Comment by BenFradet** [comment]
Thanks a lot for your continued contribution :+1:

**Pull request merged**